deploy/charts/rook-ceph/templates/role.yaml

```
@@ -75,6 +75,16 @@ rules:
 {{- if and .Values.csi.csiAddons .Values.csi.csiAddons.enabled }}
 kind: Role
 apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cephfs-csi-nodeplugin
+  namespace: {{ .Release.Namespace }} # namespace:operator
+rules:
+  - apiGroups: ["csiaddons.openshift.io"]
+    resources: ["csiaddonsnodes"]
+    verbs: ["create"]
```

```suggestion
    verbs: ["create"]
```
don't we need other permissions?

---

AFAIK, Not really.

---

okay, can you mark the PR ready for review or anything else is required for the PR

---

Yes, was waiting on testing part. 
Now its ready for review.